healthworkforce/kubernetes/importer/volume/openhim-import.json

```
@@ -48,18 +53,18 @@
       "requestBody": true,
       "responseBody": true,
       "rewriteUrlsConfig": [],
-      "name": "Mapping mediator",
+      "name": "mCSD mediator",
+      "description": "Triggers the retrieving of data and the mapping of that data into a FHIR bundle, which will then be sent to a FHIR metadata store ",
+      "pollingSchedule": "0 * * * *",
       "urlPattern": "^.*$",
```

Lets update this patern to not be a `catch all`. Anything request that doesnt meet the other patterns will end up triggering this polling channel
```suggestion
      "urlPattern": "^/mcsd-trigger$",
```